### ISSUE TITLE
Lifetime stats not correct. Probably shows since last DB version update

### ISSUE THREAD

**Issue by SM411** [RocketMap/RocketMap#1908]
On stats page, the map lifetime does not show the full content of the DB. I noticed this by changing from last year to life time and got more spawns. I think lifetime show all pokemons since last DB update.

**Comment by SiteEffect** [comment]
True, but you don't want to show the full DB content in a lot of cases :-P

**Comment by Kneckter** [comment]
Not trying to necro an old topic but I looked into this. The values are different but there is a reason; active scanning. The values change between the time you load one view to the next. The value is also cached so it won't be updated if you switch back. You can see this if you clear you cache and load the views in reverse order to see the numbers grow (until the clean wipes some out) even though the time frame is getting smaller (until your max time frame).

![image](https://user-images.githubusercontent.com/16466177/37187067-a0286360-2316-11e8-97e3-63f88dd50abc.png)

A screenshot of my data. I only keep 48 hours worth of pokemon data. You can get the same values for all the views greater than 48 hours if you stop the scanners. My DB wiped some out while I was viewing the 1 year and lifetime while running so you can see a drop in the "While Running" column.

I don't think this is a real issue because the cached data helps lessen the server load.
